frontend/src/stores/art.ts

```diff
@@ -19,15 +20,29 @@ export const useArtStore = defineStore('art', () => {
       const res = await fetch(`http://localhost:3000/items/random?type=${type}`, requestOptions)
       if (!res.ok) throw new Error(`HTTP error! status: ${res.status}`)
       const data = await res.json()
-      console.log(data)
       return data
     } catch (error) {
       console.error('failed to fetch art 💥', error)
       return null
     }
   }
+  // tyr having backend send two images at once to prevent ispecg element network cheating
+
+  const comboCountPlus = () => {
```

Comment:
with aforementioned [storetorefs](https://pinia.vuejs.org/core-concepts/#Destructuring-from-a-Store) this would be unnecessary. if storetorefs is used should just be
`artStore.combo.value++` and `artStore.combo.value = 0`